src/disclosures/js/index.js

```diff
@@ -63,7 +63,6 @@ var app = {
             } );
         }
         // set financial caps based on data
-        financialView.setCaps( getFinancial.values() );
```

Comment:
This line needs to remain in order for the About This Tool page to display loan and grant caps.